lib/potluck.rb

```diff
@@ -0,0 +1,39 @@
+require './lib/dish'
+
+class Potluck
+
+  attr_reader :date, :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu_hash = {
```

Comment:
Think about wether you _really_ need an instance variable for this - since you are never calling it from outside the class, it could be a bit of overkill.  Not hurting anything, just something to think about.